src/spinLock.h

```diff
@@ -17,8 +17,12 @@
 #ifndef _SPINLOCK_H
 #define _SPINLOCK_H
 
+#if defined(__arm__) || defined(__thumb__)
+#include <sched.h>
+#define pause() sched_yield()
+#else
 #define x86_pause()  asm volatile("pause")
```

Comment:
In this case `x86_pause()` should be also renamed. But `pause()` is a library function defined in <unistd.h>